City-of-Helsinki/drupal-helfi-rekry
# UHF-7607: Helfi global navigation

# [UHF-7607](https://helsinkisolutionoffice.atlassian.net/browse/UHF-7607)

## What was done

Enabled global navigation as read only. Updated hdbt-theme

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-7607_enable_navigation`
  * `make fresh`
* Run `make drush-cr`

## How to test


- Check that rest module is installed & enabled & configured
  - /api/v1/global-mobile-menu should be accessible by unauthenticated & authed users
  - /api/v1/global-mobile-menu should return global navigation data json
- Go to any content page, set page to mobile size
  - Global navigation should be visible with rekry's own menu in it  




[UHF-7607]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-7607?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
